release_notes.md

```diff
@@ -1,26 +1,8 @@
 # What's New
 
-The STIX endpoint to fetch datasets from a FortiSOAR instance has been enhanced to provide the list of feeds in a CSV format as well. With this change, systems such as FortiGate can directly pull threat intel via a FortiSOAR dataset.
+- With FortiSOAR v7.3.0 and FortiGuard Connector v1.1.0, the solution pack now enables support for Threat Intel Management SKU.
+    - Enable the Threat Intel Management SKU for best-in-class FortiGuard's threat intel feeds.
+    - The feeds from Threat Intel Management SKU have a rich variety of data, ranging from IPs, URLs, Domain and malicious hashes carefully curated by our team of experts.
+    - The entire feeds database is labelled with the relevant threat types, and associated LockHeed Martin Kill Chain Phases, that enables user with a lot of contextual information to understand the nature of threat.
 
-## Threat Intel Feed Enhancements
-
-The **Threat Intel Feeds** module better tracks the date and time at which the feeds were ingested or updated into the FortiSOAR instance.
```

Comment:
Removed content is a good information, whether its also a part of Usage or content md file?

Reply:
The removed content was part of the last release notes